packages/devtools_app/lib/src/framework/initializer.dart

```diff
@@ -167,7 +167,7 @@ class _InitializerState extends State<Initializer>
           child: Column(
             children: [
               const Spacer(),
-              Text('Disconnected', style: theme.textTheme.headline3),
```

Comment:
should this change to `headlineMedium`

Reply:
nope, displaySmall:
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/text_theme.dart#L307